cfpb/capital-framework
# Adds cf-notification docs

## Additions

- Adds cf-notification to docs site.

## Testing

- Run `npm run cf-link` in `canary` branch.
- Pull this branch down with `git fetch && git checkout gh-pages-notifications`
- Run `npm run build && npm start`
- Check http://localhost:3000/components/cf-notification/
